client/src/components/Dashboard.jsx

```diff
@@ -1,11 +1,10 @@
 import React, { useState, useEffect } from 'react';
-import './workout-list.css';
-import '../index.css';
-import MediaContainer from './media-container';
+import './Dashboard.css';
+import MediaContainer from './MediaContainer';
 import MeatballMenu from './meatball-menu';
 
 
-const NextPage = () => {
+const Dashboard = () => {
```

Comment:
this could probably be renamed to like "Feed" in the future or something similar!